SU-HSDO/suhumsci
# [STN-62] Scaffold templates/patterns for HumSci Base

# READY FOR REVIEW

# Summary

- Added the Drupal templates and UI Patterns from the legacy HumSci theme to support all current features.
- Also added configuration for Colorful theme block placement and configuration.
- re-organized the patterns folder to be a flat structure (no more atoms, molecules, organisms)
- Added more Drupal templates from Stanford Basic (blocks, menus, page title, html, node, page, region)
- Added favicon and screenshot (for Themes/Appearance UI)
- Removed Highlighted Region from theme (based on how the Stanford Basic theme is setup)
- Removed Google Analytics tag manager

# Steps to Test

If using Lando, prefix drush commands with `lando`.

1. Import configuration (`drush cim`) 
1. Clear caches (`drush cr`)
1. Enable the Colorful theme (`drush theme:enable humsci_colorful`)
1. Ensure various pages render without any errors
1. Compare Blocks placement on the Colorful theme to that of the legacy HumSci theme. Ensure they match.
1. Check `/patterns` page (that it works, but appears unstyled)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
